examples/basic-setup/webpack.skpm.config.js

```diff
@@ -1,11 +1,13 @@
-module.exports = config => {
-  config.node = {
-    console: false,
-    global: false,
-    process: true,
-    __filename: 'mock',
-    __dirname: 'mock',
-    Buffer: false,
-    setImmediate: false,
-  };
+const path = require('path');
+
+module.exports = (config) => {
+  if (process.env.LOCAL_DEV) {
+    config.resolve = {
+      ...config.resolve,
+      alias: {
+        ...config.resolve.alias,
+        'react-sketchapp': path.resolve(__dirname, '../../lib/'), // FIXME: Should be able to do '../../', as that's a close simulation of an 'npm install' and import 'react-sketchapp'
```

Comment:
Left in this comment to note that `'../../'` should be used instead of `'../../lib/'`, but it is failing due to using the `lib/module` entrypoint from the `package.json.